lib/credo/check/warning/dbg.ex

```diff
@@ -0,0 +1,49 @@
+defmodule Credo.Check.Warning.Dbg do
+  use Credo.Check,
+    base_priority: :high,
+    explanations: [
+      check: """
+      This check warns about calls to the dbg/3 macro
+      or the dbg/2 Kernel fn.
```

Comment:
Nitpicking for good measure: Let's change this to

```suggestion
      Calls to dbg/2 and dbg/3 should mostly be used during debugging sessions.

      This check warns about those calls, because they probably have been committed
      in error.
```